src/actions/remove/remove.factory.js

```diff
@@ -1,10 +1,11 @@
 module.exports = (eventExists, mapUsernameToUserInfo, REMOVE) => {
   return (parsedCommand, usernameToIds, myUserId, events) => {
-    const { event, usernames } = parsedCommand;
-    eventExists(event, events);
+    console.log(parsedCommand);
+    const { from, usernames } = parsedCommand;
+    eventExists(from, events);
     return {
       type: REMOVE,
-      event,
+      event: from,
       userInfos: mapUsernameToUserInfo(usernames, usernameToIds, myUserId)
     };
   };
```

Comment:
Remove this file from the pull request. You can make a separate pull request for this.

Reply:
Same goes to all other modified files that are not part of the docs.